hw/ip_templates/alert_handler/data/alert_handler.prj.hjson

```diff
@@ -11,7 +11,7 @@
     version:            "1.0",
     life_stage:         "L1",
     design_stage:       "D2",
-    verification_stage: "V1",
+    verification_stage: "V2",
```

Comment:
Thanks everyone for attending the review meeting.
Here is the [notes](https://docs.google.com/document/d/1s9lnLFccW0Y479oxogVnm2-I3HgPlt7UcbqJXHdTlps/edit?resourcekey=0-h91JxZynFGWCsOZB01qfMQ) from the meeting.

The TODOs are listed below:

- [ ] Check with security group if missing a recoverable alert during alert_init is a problem
        If so, add a V3 test: sending a alert_init while receiving recoverable alert 

      Added a V3 item: https://github.com/lowRISC/opentitan/pull/9723 

- [ ] Check coverage is converging
![image](https://user-images.githubusercontent.com/11466553/146254369-255e4076-df10-439e-941d-05424012ed54.png)

- [ ] Add an assertion to check the alert/esc counter is saturated after reaching the maximum value. Also check if it’s converged in FPV (we can change the parameter to reduce the size for ping)

   Added assertion: https://github.com/lowRISC/opentitan/pull/9715 

- [ ] Do the regwen coverage in RAL model

    Filed issue: https://github.com/lowRISC/opentitan/issues/9673 

- [ ] Check if the checklist has any missing item

   Checked and no missing item found!

- [ ] Check if we can remove exclusion at *_clear_regwen

  Done with PR: https://github.com/lowRISC/opentitan/pull/9674

Reply:
This checklist is awesome. Thanks